src/shared/components/palette/datetime-picker/datetime-picker.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, ElementRef, Inject, Input, OnInit, ViewChild, ViewEncapsulation } from '@angular/core';
-import { FormControl } from '@angular/forms';
+import { FormControl, Validators } from '@angular/forms';
```

Comment:
I'm a bit nervous about this component at the moment as it looks like there were extensive changes on both `master` and `release-4.1.0` so it needs more investigation... and will probably need some regression testing.